packages/components/src/components/DaCardPost.vue

```diff
@@ -8,7 +8,8 @@
       <img class="card__footer__icon lazyload"
```

Comment:
It's better to wrap the `img` element inside a `button` element for accessibility reasons. What do you think?

Reply:
Yup that makes sense!